lib/spectcl.js

```diff
@@ -327,7 +330,7 @@ Spectcl.prototype.expect = function(expArr, cb){
 
           if(expectationCb){
               debug('[expect] [%d] calling handler for %s', self.child.pid, matchedExpectation)
-              cbRetVal = expectationCb(matchedExpectation, finalCb)
+              cbRetVal = expectationCb(matchedExpectation, finalCb, expectMatch[matchedExpectation])
```

Comment:
I'd prefer to have the callback continue to be at the end of the parameter list rather than tacking the additional argument on to the end.  This would require modifications to the expectation callbacks and be a breaking change, but I feel it would be cleaner than tacking the new field on to the end parameter list.